test/integration/driver.go

```diff
@@ -61,7 +61,7 @@ const (
 	caImage = "gcr.io/istio-testing/istio-ca:2baec6baacecbd516ea0880573b6fc3cd5736739"
 
 	// mixerImage specifies the default mixer docker image used for e2e testing *update manually*
-	mixerImage = "gcr.io/istio-testing/mixer:49e721e15d481cd5d92d9a2b30b5e8fcdcafdb63"
+	mixerImage = "gcr.io/istio-testing/mixer:986c6e1eb9c7dcff42e3df5aefa9ce858ece885e"
```

Comment:
this is not a minor change, don't bundle the subtle things like this

Reply:
We need the pluralization from mixer.

Reply:
It's not subtle, it passed pilot e2e tests, it can be upgraded.

Reply:
It it were easier to get a change committed, we would not end up bundling multiple fixes in one PR.